lib/replicache.dart

```diff
@@ -409,7 +409,28 @@ class Replicache implements ReadTransaction {
     }
   }
 
-  /// Registers a function to be used for write transactions. This function is
+  /// Registers a *mutator*, which is used to make changes to the data.
+  ///
+  /// Replays
```

Comment:
I'm not actually sure about the formatting here. I don't know what syntax Flutter uses. Also can you fill in the links below.

Reply:
Client view can link to the relevant section of the quickstart on the replicache repo for now.